assets/training/distillation/components/data_generation/spec.yaml

```diff
@@ -83,8 +83,20 @@ inputs:
 
   enable_chain_of_thought:
     type: string
-    default: "true"
+    default: "false"
     description: Enable Chain of thought for data generation
+  
+  data_generation_task_type:
+    type: string
+    enum:
+      - NLI
+      - CONVERSATION
+      - NLU_QA
+    description: >
+      Data generation task type. Supported values are:
+      1. NLI: Generate NLI data
+      2. CONVERSATION: Generate conversational data (multi/single)
+      4. NLU_QUESTION_ANSWERING: Generate QnA data
```

Comment:
Keep the key as in enum, and explaination should have full form.

Reply:
Ok

Reply:
Done